src/system/CategoriesModule/Form/Type/ConfigType.php

```diff
@@ -121,7 +122,7 @@ private function getCategoryChoices()
         $category = CategoryUtil::getCategoryByID(1);
         $categoryList = CategoryUtil::getSubCategoriesForCategory($category, $recurse, $relative, $includeRoot, $includeLeaf, $all, '', '', null, 'sort_value');
 
-        $lang = ZLanguage::getLanguageCode();
+        $lang = !empty($request) ? $request->getLocale() : 'en';
```

Comment:
in this Form class, you need to set a default value for the arg (in `setDefaults()`) so you can set the `en` value there.